src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-cognito-monitoring-integration.mdx

```diff
@@ -11,6 +11,10 @@ redirects:
   - /docs/aws-aws_cognito
 ---
 
+<Callout variant="important">
+  Enable the [AWS CloudWatch Metric Streams integration](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-metric-stream/) to monitor all CloudWatch metrics from your AWS services, including custom namespaces. Individual integrations are no longer our recommended option.
```

Comment:
I'd rephrase the last sentence to avoid passive voice. Something like, "We don't recommend installing individual integrations."
Could we also add context about why we don't recommend it anymore?

Reply:
Thanks for the feedback. However, the PM wants this particular wording ;)